README.md

```diff
@@ -207,17 +207,16 @@ First compile the cert-manager E2E test binary as follows:
 
 ```sh
 cd projects/cert-manager/cert-manager
-bazel build //test/e2e:e2e
+make e2e-build
```

Comment:
cert-manager no longer uses bazel and I've added an equivalent make target in: 
 * https://github.com/cert-manager/cert-manager/pull/5804